esphome/components/pcd8544/display.py

```diff
@@ -17,6 +17,7 @@
     cv.Required(CONF_DC_PIN): pins.gpio_output_pin_schema,
     cv.Required(CONF_RESET_PIN): pins.gpio_output_pin_schema,
     cv.Required(CONF_CS_PIN): pins.gpio_output_pin_schema,  # CE
+    cv.Optional(CONF_CONTRAST, 0x7f): cv.int_,
```

Comment:
We always specify `default=` to make it more readable
```suggestion
    cv.Optional(CONF_CONTRAST, default=0x7f): cv.int_,
```